ocaml/xapi/xapi_pool.ml

```diff
@@ -3952,3 +3952,31 @@ let put_bundle_handler (req : Request.t) s _ =
       | None ->
           ()
   )
+
+let ssh_helper ~__context ~action ~error =
```

Comment:
The name does not reflect what the function does; it is not specific to SSH. In general, you could consider group SSH functions into an SSH sub module.